backend/apid/middlewares/authorization.go

```diff
@@ -1,10 +1,15 @@
 package middlewares
 
 import (
+	"context"
 	"net/http"
 
 	"github.com/sensu/sensu-go/backend/apid/actions"
 	"github.com/sensu/sensu-go/backend/authorization"
+	"github.com/sensu/sensu-go/backend/authorization/rbac"
+	"github.com/sensu/sensu-go/backend/store"
+	"github.com/sensu/sensu-go/transport"
+	corev2 "github.com/sensu/sensu-go/types"
```

Comment:
Could we replace `github.com/sensu/sensu-go/types` with `github.com/sensu/sensu-go/api/core/v2`?